src/ripple/app/misc/Manifest.h

```diff
@@ -257,6 +257,32 @@ class ManifestCache
     PublicKey
     getMasterKey (PublicKey const& pk) const;
 
+    /** Returns master key's current manifest sequence.
+
+        @return sequence corresponding to Master public key
+
+        @return 0 if no known sequence from a manifest
+
+        @par Thread Safety
+
+        May be called concurrently
+    */
+    std::uint32_t
+    getSequence (PublicKey const& pk) const;
+
+    /** Returns master key's current domain
```

Comment:
Not to get into a grammar war, but this possessive is likely to irritate someone at some point. It also reads slightly weird, imo. Consider:

`/** Return the domain claimed by a given public key

Reply:
Done